system/jlib/jtrace.cpp

```diff
@@ -59,6 +61,35 @@ namespace opentel_trace = opentelemetry::trace;
 
 using namespace ln_uid;
 
+
+class CustomOTELLogHandler : public opentelemetry::sdk::common::internal_log::LogHandler
+{
+    void Handle(opentelemetry::sdk::common::internal_log::LogLevel level,const char * file, int line, const char * msg,
+                const opentelemetry::sdk::common::AttributeMap &attributes) noexcept override
+    {
+        if (!isEmptyString(msg))
+        {
+            LogMsgCategory logCategory = MCdebugInfo;
+            switch(level)
+            {
+                case opentelemetry::sdk::common::internal_log::LogLevel::Error:
```

Comment:
indentation of the case statements (or the contents) is wrong and hard to read.

Reply:
Indented switch case contents and now using const LogMsgCategory *

Reply:
thoughts on choosing the preferred format and automatically enforcing it via github action?